content/docs/howto/whitelist_assets.md

```diff
@@ -0,0 +1,42 @@
+---
+weight: 1
+bookFlatSection: true
+---
+
+# Whitelist your asset
+
+Terraswap aims to be a pure DEX, that anyone can mint & list their own assets. On the contrary of that, many duplicated scams are also listed and it harms to many innocent users, especially crypto newbies. As this matter can harm to grow our ecosystem, we adopt the whitelisting system of Terra repository.
+
+Therefore, for project developers, if you want to unify your asset on Terraswap with your logo, you need to whitelist your asset information on the Terra Asset repository.
+
+---
+> **NOTE**
+>
+> As Terraswap aims & is a pure DEX, Terraswap does not judge whether the asset is a genuine or scam. All assets should be treated equally & all users have responsible for their transactions on Terraswap. Terraswap team has been exempted from any loss from its app usage. Be sure that all users are trading the correct token, always check the address before buying.
+---
+
+## Find Terra Asset repository
+
+Please find it on [`assets` repository](https://github.com/terra-money/assets)
+
+## Add your asset information
+
+1. If your asset is Terra native
+    - Modify `cw20/token.js`
+    - Add protocol, symbol, name, token contract address, the path of the icon, and decimals
+    - [Example](https://github.com/terra-money/assets/pull/99)
```

Comment:
This pr is in old json format. It would be better to change it to js format pr

Reply:
https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/31/commits/bc8900ea03499051b8195735f96ac683fb8de407